src/lxc/tools/lxc_top.c

```diff
@@ -314,21 +320,53 @@ static void stat_get_blk_stats(struct lxc_container *c, const char *item,
 	return;
 }
 
+static void try_cgroup2(struct lxc_container *c, u_int64_t *stat, const char* path_cgroup1, const char* path_cgroup2, 
+	const char* match, bool call_match) {
+	
+	int ret_cgroup2; 
+	
+	if (call_match) {
+		ret_cgroup2 = stat_match_get_int(c, path_cgroup2, match, 1); 
+		if (ret_cgroup2 < 0) {
+			*stat = stat_match_get_int(c, path_cgroup1, match, 1);
+		}
+		
+	} else {
+		ret_cgroup2 = stat_get_int(c, path_cgroup2); 
+		
+		if (ret_cgroup2 < 0) {
+			*stat = stat_get_int(c, path_cgroup1);
+		} else {
+			*stat = ret_cgroup2; 
+		}
+	}
+}
+
```

Comment:
We can easily [avoid using boolean argument](https://google.com/search?q=avoid+boolean+arguments) here.

```suggestion
static void try_match_cgroup2(struct lxc_container *c, u_int64_t *stat, const char* path_cgroup1, const char* path_cgroup2, 
	const char* match) {
	
	int ret_cgroup2; 
	ret_cgroup2 = stat_match_get_int(c, path_cgroup2, match, 1); 
	if (ret_cgroup2 < 0) {
		*stat = stat_match_get_int(c, path_cgroup1, match, 1);
	}
}		

static void try_cgroup2(struct lxc_container *c, u_int64_t *stat, const char* path_cgroup1, const char* path_cgroup2) {
	ret_cgroup2 = stat_get_int(c, path_cgroup2); 
		
	if (ret_cgroup2 < 0) {
		*stat = stat_get_int(c, path_cgroup1);
	} else {
		*stat = ret_cgroup2; 
	}
}

```